modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/IGridClusterStateProcessor.java

```diff
@@ -108,30 +108,20 @@ boolean onStateChangeMessage(AffinityTopologyVersion topVer,
     /** */
     void cacheProcessorStarted();
 
-    /**
-     * @param activate New cluster state.
-     * @param baselineNodes New baseline nodes.
-     * @param forceChangeBaselineTopology Force change baseline topology.
-     * @return State change future.
-     * @deprecated Use {@link #changeGlobalState(ClusterState, Collection, boolean)} instead.
-     */
-    @Deprecated
-    IgniteInternalFuture<?> changeGlobalState(
```

Comment:
Why this deletion?

Reply:
> Why this deletion?

Same as above ("To simplify and reduce internal API and code base...")